devices/families/sdm845-mainline/kernel/config.aarch64

```diff
@@ -2453,10 +2536,8 @@ CONFIG_TOUCHSCREEN_ELAN=y
 # CONFIG_TOUCHSCREEN_MTOUCH is not set
 # CONFIG_TOUCHSCREEN_NOVATEK_NVT_TS is not set
```

Comment:
For support with xiaomi-beryllium (#779) `CONFIG_TOUCHSCREEN_NOVATEK_NVT_TS` has to be set to `y`.